webhook_to_fedora_messaging/config/__init__.py

```diff
@@ -2,19 +2,44 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from os import environ
 
-import tomllib
+import logging
+import os
+from logging.config import dictConfig
+from pathlib import Path
 
-from webhook_to_fedora_messaging.exceptions import ConfigError
+from sqlalchemy_helpers.fastapi import AsyncDatabaseManager
+
+from webhook_to_fedora_messaging import database
+from webhook_to_fedora_messaging.config import standard
 
 
 def get_config() -> dict:
-    path = environ["W2FM_APPCONFIG"]
+    path = os.getenv("W2FM_CONFIG")
```

Comment:
I like this change, but you must also change other files where it's defined. `git grep APPCONFIG` will tell you which ones :-)